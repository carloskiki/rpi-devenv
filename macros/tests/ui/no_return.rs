#[macros::main]
fn kernel_main() {}

fn main() {}
