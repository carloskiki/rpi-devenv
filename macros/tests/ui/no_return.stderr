error: main function must never return (`-> !`)
 --> tests/ui/no_return.rs:1:1
  |
1 | #[macros::main]
  | ^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `macros::main` (in Nightly builds, run with -Z macro-backtrace for more info)
